### ISSUE TITLE
Option language does not work for Microsoft Edge TTS

### ISSUE THREAD

**Issue by jonalbr** [nimroddolev/chime_tts#79]
### Checklist

- [X] I have filled out the template to the best of my ability.
- [X] This only contains 1 feature request (if you have multiple feature requests, open one feature request for each feature request).
- [X] This issue is not a duplicate feature request of [previous feature requests](https://github.com/nimroddolev/chime_tts/issues?q=is%3Aissue+label%3A%22Feature+Request%22+).

### Is your feature request related to a problem? Please describe.

Hi, if using `edge_tts` with the parameter `language` the voice stays the default (for me `de-DE`) voice and does not adapt to the chosen voice/language.

```
service: chime_tts.say
target:
  entity_id: media_player.arbeitszimmer
data:
  chime_path: bells
  end_chime_path: ""
  offset: 450
  final_delay: 0
  volume_level: 0.35
  message: Das ist ein Test.
  tts_platform: edge_tts
  language: de-DE-AmalaNeural
```

I greatly appreciate your help.
Kind regards,
Jonas

### Describe the solution you'd like

The parameter 'language' does change the voice as desired.

### Describe alternatives you've considered

With the following service provided by Home Assistant / the Edge TTS integration, choosing the language is working:
```
service: tts.edge_tts_say
data:
  entity_id: media_player.arbeitszimmer
  language: de-DE-AmalaNeural
  message: Das ist ein Test.
```

### Additional context

.

**Comment by nimroddolev** [comment]
Hi @jonalbr, thank you for the pull request! I've released a new beta version, v0.13.1-beta2, which includes your commit. Could you please upgrade to the new version and let me know if the `language` parameter is now working for you with the Microsoft Edge TTS platform?

To install the new beta version:
1. Open the <a href="https://my.home-assistant.io/redirect/hacs_repository/?owner=nimroddolev&repository=chime_tts&category=integration" title="Chime TTS HACS repository" target="_blank">Chime TTS repository in HACS</a>
1. Click on the `⋮` button in the top right
1. Select the `↻ Redownload` option
1. Enable the `Show beta versions` option (it may take a moment to update the list)
1. Select version `v0.13.1-beta2` from the updated version drop-down list
1. Restart Home Assistant

**Comment by jonalbr** [comment]
I can confirm that the issue is fixed for me in `v0.13.1-beta2 `.

**Comment by nimroddolev** [comment]
Thank you Jonas. The fix will be included in the next release.
